============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-8.3.3, pluggy-1.5.0
rootdir: /home/wilkinson/Documents/TCC/TG
collected 3 items

QuixBugs/python_testcases/test_minimum_spanning_tree.py .F.              [100%]

=================================== FAILURES ===================================
____________________________________ test2 _____________________________________

    def test2():
        """Case 2: Strongly connected tree input.
        Output: (2, 5) (1, 3) (2, 3) (4, 6) (3, 6)
        """
    
        result = minimum_spanning_tree(
            {
                (1, 2): 6,
                (1, 3): 1,
                (1, 4): 5,
                (2, 3): 5,
                (2, 5): 3,
                (3, 4): 5,
                (3, 5): 6,
                (3, 6): 4,
                (4, 6): 2,
                (5, 6): 6,
            }
        )
    
>       assert result == {(2, 5), (1, 3), (2, 3), (4, 6), (3, 6)}
E       assert {(1, 3), (2, ..., (4, 6), ...} == {(1, 3), (2, ...3, 6), (4, 6)}
E         
E         Extra items in the left set:
E         (5, 6)
E         (3, 5)
E         Use -v to get more diff

QuixBugs/python_testcases/test_minimum_spanning_tree.py:46: AssertionError
=========================== short test summary info ============================
FAILED QuixBugs/python_testcases/test_minimum_spanning_tree.py::test2 - asser...
========================= 1 failed, 2 passed in 0.04s ==========================
Different AST

prover solução mas insere novo bug
